.gitignore

```diff
@@ -4,3 +4,4 @@ _site
 /gojekyll
 /gojekyll-race
 coverage.out
+go.sum
```

Comment:
My reading of https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions is that `go.sum` should go in the repo (if `go.mod` does). That seems like a good idea to make sure that all developers and the CI are using the same versions of dependent modules.